index.html

```diff
@@ -67,27 +67,27 @@ <h1 class="about">
       </div>
       <ul class="social-media-container d-flex">
         <li class="element">
-          <a class="social twitter" href="#"><img src="./images/twitter.svg" alt=""></a>
+          <a class="social twitter" href="#"><img src="./images/twitter.svg" alt="Isai's twitter"></a>
         </li>
         <li class="element">
-          <a class="social github" href="#"><img src="./images/linkedin.svg" alt=""></a>
+          <a class="social github" href="#"><img src="./images/linkedin.svg" alt="Isai's linkedin"></a>
         </li>
         <li class="element">
-          <a class="social" href="#"><img src="./images/mail.svg" alt=""></a>
+          <a class="social" href="#"><img src="./images/mail.svg" alt="Isai's mail"></a>
         </li>
         <li class="element">
-          <a class="social mail" href="#"><img src="./images/github.svg" alt=""></a>
+          <a class="social mail" href="#"><img src="./images/github.svg" alt="Isai's github"></a>
         </li>
         <li class="element">
-          <a class="social Linkedin" href="#"><img src="./images/other.svg" alt=""></a>
+          <a class="social Linkedin" href="#"><img src="./images/other.svg" alt="Isai's other media"></a>
         </li>
       </ul>
     </div>
   </section>
   <section class="work-section-container d-grid">
     <div class="work-card desk-d-grid d-flex ja-center">
       <div class="image-card">
-        <img src="./images/work-4.png" alt="" class="image">
+        <img src="./images/work-4.png" alt="project example" class="image">
       </div>
       <div class="text-card d-flex">
         <div class="primary-text">
```

Comment:
OPTIONAL

- [ ] For each project title, enclose it within a header tag such as `h4` or `h5` instead of a `p` tag.